src/Core/Form/ChoiceProvider/CurrencyByIdChoiceProvider.php

```diff
@@ -27,12 +27,13 @@
 namespace PrestaShop\PrestaShop\Core\Form\ChoiceProvider;
 
 use PrestaShop\PrestaShop\Adapter\Currency\CurrencyDataProvider;
+use PrestaShop\PrestaShop\Core\Form\FormChoiceAttributeProviderInterface;
 use PrestaShop\PrestaShop\Core\Form\FormChoiceProviderInterface;
 
 /**
  * Class CurrencyByIdChoiceProvider provides currency choices with ID values.
  */
-final class CurrencyByIdChoiceProvider implements FormChoiceProviderInterface
+final class CurrencyByIdChoiceProvider implements FormChoiceProviderInterface, FormChoiceAttributeProviderInterface
```

Comment:
I guess adding an interface on a final class is not a Breaking Change

Reply:
Modifying a final class is not a BC because it's not extendable.